packages/display/src/Container.js

```diff
@@ -318,12 +318,13 @@ export default class Container extends DisplayObject
      */
     updateTransform()
     {
-        this._boundsID++;
-
-        this.transform.updateTransform(this.parent.transform);
+        const parent = this.parent || this._tempDisplayObjectParent;
```

Comment:
Note that the original reason we did it the way we did was that this branch here caused significant performance issues. That may have changed recently, but I wanted to call out the context for why we did it a more convoluted way.

Reply:
Actually, that's a good idea, we can make parent a property :)